website/docs/apis/java-client.md

```diff
@@ -0,0 +1,292 @@
+---
+sidebar_label: "Java Client"
+sidebar_position: 1
+---
+
+<!--
+ Copyright (c) 2025 Alibaba Group Holding Ltd.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-->
+
+# Fluss  Java Client
+## Overview
```

Comment:
We need to introduce how to add the Java client maven dependency. And introduce what APIs provided. Overview, Fluss provides `Admin` API and `Table` API, we can first introduce what the API mainly used for.